components/dashboard/src/projects/Prebuilds.tsx

```diff
@@ -228,4 +229,63 @@ export function prebuildStatusIcon(status: PrebuiltWorkspaceState | undefined) {
         default:
             break;
     }
+}
+
+export function PrebuildInstanceStatus(props: { prebuildInstance?: WorkspaceInstance, isDark?: boolean }) {
+    let status = <></>;
+    let details = <></>;
+    switch (props.prebuildInstance?.status.phase) {
+        case undefined: // Fall through
+        case 'unknown':
+            status = <div className="flex space-x-1 items-center text-yellow-600">
+                <img className="h-4 w-4" src={StatusPaused} />
+                <span>PENDING</span>
+                </div>;
+            details = <div className="flex space-x-1 items-center text-gray-400">
+                <img className="h-4 w-4 animate-spin" src={props.isDark ? SpinnerDark : Spinner} />
+                <span>Prebuild in progress ...</span>
+                </div>;
+            break;
+        case 'preparing': // Fall through
+        case 'pending': // Fall through
+        case 'creating': // Fall through
+        case 'initializing': // Fall  through
+        case 'running': // Fall through
+        case 'interrupted':
+            status = <div className="flex space-x-1 items-center text-blue-600">
+                <img className="h-4 w-4" src={StatusRunning} />
+                <span>RUNNING</span>
+                </div>;
+            details = <div className="flex space-x-1 items-center text-gray-400">
+                <img className="h-4 w-4 animate-spin" src={props.isDark ? SpinnerDark : Spinner} />
+                <span>Prebuild in progress ...</span>
+                </div>;
+            break;
+        case 'stopping': // Fall through
+        case 'stopped':
+            status = <div className="flex space-x-1 items-center text-green-600">
+                <img className="h-4 w-4" src={StatusDone} />
+                <span>READY</span>
+                </div>;
+            details = <div className="flex space-x-1 items-center text-gray-400">
+                <img className="h-4 w-4 filter-grayscale" src={StatusRunning} />
+                <span>{!!props.prebuildInstance?.stoppedTime
```

Comment:
**issue**: This is currently showing duration as seconds like `5s` but this could look confusing when showing time duration longer than a minute. For example, `306s`. Ideally this could surface time duration for `306s` as like `05:06` as described in the initial specs in https://github.com/gitpod-io/gitpod/issues/4948#issuecomment-895927769. However, it should be fine to resolve this in a follow up issue. 🏓 

| 5s | 306s |
|-|-|
| ![duration-1](https://user-images.githubusercontent.com/120486/131980979-5c977e94-4555-4ad0-8e1b-b915b0e15353.png) | ![duration-2](https://user-images.githubusercontent.com/120486/131980984-8d9670df-54bb-40cc-a304-90c23bae36ed.png) |

Reply:
Filed as https://github.com/gitpod-io/gitpod/issues/5525